pkg/operator/workloads/api_workload.go

```diff
@@ -463,11 +486,14 @@ func serviceSpec(ctx *context.Context, api *context.API) *kcore.Service {
 			"appName":      ctx.App.Name,
 			"workloadType": workloadTypeAPI,
 			"apiName":      api.Name,
+			"service":      internalAPIName(api.Name, ctx.App.Name),
+			"app":          internalAPIName(api.Name, ctx.App.Name),
```

Comment:
Is `app` label necessary? (also line 496)

Reply:
yes